webpack.config.js

```diff
@@ -1,9 +1,15 @@
+const webpack = require('webpack');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const path = require('path');
-const ExtractCssChunks = require('extract-css-chunks-webpack-plugin');
 
+const mode = process.env.NODE_ENV || 'development';
 module.exports = {
-  mode: process.env.NODE_ENV || 'development',
+  mode,
   entry: './src/index.js',
+  // In production, always be IE11 compatible. In dev, be IE11 compatible
+  // if IE11=1 is in the environment, as this currently breaks hot reload:
+  // https://github.com/webpack/webpack-dev-server/issues/2758
```

Comment:
We don't have webpack-style hot reload yet in a3-demo, but we're going to want it, and we want the other benefits of the `web` target, etc. so it made sense to bring this over from assembly.